service_management/test/services/analysis/SampleObjectServiceSpec.scala

```diff
@@ -103,5 +103,36 @@ class SampleObjectServiceSpec
           fail(s"The list contained ${sce.size} elements, expected 1.")
       }
     }
+
+    "successfully add a new sample object with status 'Degraded' " in {
+      val so = SampleObject(
```

Comment:
I tried it, but it would come out with a message that said that degraded was not an intact...so therefore this solution....